packages/wdio-spec-reporter/src/index.ts

```diff
@@ -45,6 +50,16 @@ export default class SpecReporter extends WDIOReporter {
         this._sauceLabsSharableLinks = 'sauceLabsSharableLinks' in options
             ? options.sauceLabsSharableLinks as boolean
             : this._sauceLabsSharableLinks
+        let processObj:any = process
```

Comment:
```suggestion
        let processObj = process
```

Reply:
if i remove any , i get 
![image](https://user-images.githubusercontent.com/45095911/125657702-6cb45f2b-0779-483f-8bc4-12c8d264ed00.png)

Reply:
@christian-bromann  could you have a look at this , what type should i give

Reply:
@christian-bromann  If no changes needs to be done for this comment then could you retrigger the workflow it looks like an intermittent failure. sleep(100) was 99 something, Thank you so much for your time and detailed reviews learned a lot